controller/model/api_session_model.go

```diff
@@ -87,7 +87,7 @@ func (entity *ApiSession) fillFrom(handler Handler, tx *bbolt.Tx, boltEntity bol
 	entity.MfaComplete = boltApiSession.MfaComplete
 	entity.ExpiresAt = entity.UpdatedAt.Add(handler.GetEnv().GetConfig().Api.SessionTimeoutSeconds)
 	entity.ExpirationDuration = handler.GetEnv().GetConfig().Api.SessionTimeoutSeconds
-	entity.LastActivityAt = boltApiSession.LastActivityAt
+	entity.CachedLastActivityAt = boltApiSession.LastActivityAt
```

Comment:
please verify that this is correct @andrewpmartinez ? took a stab at this but not sure if i did it properly